goldens/foo/lib/metadata.analyzer.json

```diff
@@ -55,20 +55,26 @@
                   },
                   "arguments": [
                     {
-                      "name": "aBool",
-                      "expression": {
-                        "type": "BooleanLiteral",
-                        "value": {
-                          "text": "true"
+                      "type": "NamedArgument",
+                      "value": {
+                        "name": "aBool",
+                        "expression": {
+                          "type": "BooleanLiteral",
+                          "value": {
+                            "text": "true"
+                          }
                         }
                       }
                     },
                     {
-                      "name": "anInt",
-                      "expression": {
-                        "type": "IntegerLiteral",
-                        "value": {
-                          "text": "23"
+                      "type": "NamedArgument",
+                      "value": {
+                        "name": "anInt",
+                        "expression": {
+                          "type": "IntegerLiteral",
+                          "value": {
+                            "text": "23"
```

Comment:
I've update the `BooleanLiteral`, `IntegerLiteral` and `DoubleLiteral` to hold their primitive `value` where possible (ints are a bit weird). It might be good to pass that along - in particular since evaluation (coming shortly) will only provide the `value` and not the `text`.